.github/workflows/ci-nightly.yml

```diff
@@ -26,8 +28,7 @@ jobs:
       run: |
         git config user.name ${{ env.GITHUB_PUBLISH_USER }}
         git config user.email ${{ env.GITHUB_PUBLISH_EMAIL }}
-        git remote set-url origin https://${{GITHUB_SERVICE_PAT}}:${{env.GITHUB_SERVICE_PAT}}@github.com/microsoft/fast.git
```

Comment:
I think we're missing a "env." before GITHUB_SERVICE_PAT.

Reply:
where that one comes from. It's not not on the list of the secrets we have